explore/src/main/scala/explore/cache/ProgramCacheController.scala

```diff
@@ -132,12 +134,28 @@ object ProgramCacheController
         offset =>
           ProgramSummaryQueriesGQL
             .AllProgramObservations[IO]
-            .query(props.programId.toWhereObservation, offset.orUnassign),
+            .query(props.programId.toWhereObservation, offset.orUnassign)(using
+              ErrorPolicy.IgnoreOnData
```

Comment:
Why is IgnoreOnData necessary?

Reply:
I'm trying to remember. I'll get back to you shortly...

Reply:
It's because we now get some errors with the observations for some things.
<img width="543" alt="image" src="https://github.com/user-attachments/assets/4a9dd81b-8d95-4a74-9361-e69c706ed2d1">
I guess we could review the ODB and see if that is really necessary. But, that is the way it is now,

Reply:
Yeah, that one at least sounds more like a validation message than a query error.

But I agree that we want the IgnoreOnData in the presence of this. Maybe add a comment?